lib/honeycomb/propagation/w3c.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require "base64"
+require "json"
+require "uri"
```

Comment:
Are any of these require things used in this file?

Reply:
uhhh nope. Good catch :)